src/Text.js

```diff
@@ -65,5 +65,6 @@ Text.propTypes = {
 
 Text.span = Text.withComponent('span')
 Text.p = Text.withComponent('p')
+Text.s = Text.withComponent('s')
```

Comment:
Can the term `s` be a little bit verbose to be, perhaps, `strike`?

Reply:
I like the convention of the name matching the tag.  Takes the guesswork out of the naming.  I wouldn't want it to be confused with the obsolete `<strike>` element either

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strike